doc/release-notes-26471.md

```
@@ -0,0 +1,12 @@
+Updated settings
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)